src/store/Controller.h

```diff
@@ -149,7 +149,7 @@ class Controller: public Storage
     void checkTransients(const StoreEntry &) const;
     void checkFoundCandidate(const StoreEntry &) const;
 
-    Disks *swapDir; ///< summary view of all disk caches
+    Disks *disks; ///< summary view of all disk caches (including none); never nil
```

Comment:
> Alex: The `swapDir` name itself is a bug. The correct name should have been something like `disks` or `cacheDirs`. Fixing that naming bug may not be worth the noise that fix will create and can be viewed as outside this PR scope IMO.

FWIW, I am still not sure the renaming noise is worth its benefits, but I am clearing this PR for merging. Thank you for removing all these unnecessary and misleading checks!

I have polished PR-modified code and synced PR title/description with the latest changes.